### PR TITLE
Fix or disable warnings in Python.Runtime

### PR DESCRIPTION
**Pull request by filmor**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Checklist

Check all those that are applicable and complete.

-   [ ] Make sure to include one or more tests for your change
-   [ ] If an enhancement PR, please create docs and at best an example
-   [ ] Add yourself to [`AUTHORS`](../blob/master/AUTHORS.md)
-   [ ] Updated the [`CHANGELOG`](../blob/master/CHANGELOG.md)

### CODE REVIEW

File: src/runtime/classobject.cs [hunk 1]
```diff
@@ -143,7 +144,7 @@ public override IntPtr type_subscript(IntPtr idx)
                 return Exceptions.RaiseTypeError("type(s) expected");
             }
 
-            Type gtype = AssemblyManager.LookupType($"{type.FullName}`{types.Length}");
+            Type gtype = AssemblyManager.LookupTypes($"{type.FullName}`{types.Length}").FirstOrDefault();
```
Comment (lostmsu) [inline src/runtime/classobject.cs:147]: Picking the first type of many randomly is not a good solution.

**Review by lostmsu — CHANGES_REQUESTED** [review]

File: src/runtime/classobject.cs (hunk 1 above)
Comment (filmor) [reply src/runtime/classobject.cs:147]: That's true, but this is what we are doing right now as well. I can add a follow-up ticket, but I also want to get to a warning-free build at some point s.t. we can make them fatal.

Comment (lostmsu) [reply src/runtime/classobject.cs:147]: That would do.

Comment (filmor) [reply src/runtime/classobject.cs:147]: #1320.

**Pull request merged**